PresentValueSeries/InitializeData.ipynb

```diff
@@ -67,7 +67,10 @@
         {
             "cell_type": "code",
             "source": [
-                "await Import.FromFile(\"Dimensions.xlsx\").WithFormat(ImportFormats.AocConfiguration).WithTarget(DataSource).ExecuteAsync()"
+                "await Import.FromFile(\"Dimensions.xlsx\")",
+                "\n            .WithFormat(ImportFormats.AocConfiguration)",
+                "\n            .WithTarget(DataSource)",
+                "\n            .ExecuteWithStoreActivityAsync(Session, DataSource, DataSetReader)"
```

Comment:
From an API point of view, I would separate the execution from the activity requirement.
Something like:
.WithStoreActivity(...)
.ExecuteAsync(...)

It this possible?

Reply:
Not sure about that. In order to store the activity information we need activity log of the import, which is generated by ExecuteAsync(). So I do not see a good way to bypass this step. 
What we can do is to save first the file information (for this we do not need the log result), then execute and then to write down the activity log. The API then would look something like 
.WithStoreFileAsync(Session, DataSource, DataSetReader)
.ExecutePassAsync()
.WithStoreLogAsync(Session, DataSource)

For this I would define two new wrappers to pass the File Id info from the first to the third method
public record BuilderWrapper(Guid FileId, Import OptionsBuilder builder)
public record ActitivityLogWrapper (Guid FileId, ActivityLog log). Not sure if that would be an improvement. We can talk about this when you have more time.